endpoints/api/repositorynotification_models_interface.py

```diff
@@ -9,6 +9,7 @@ class RepositoryNotification(
     namedtuple(
         "RepositoryNotification",
         [
```

Comment:
is this the DB ID? we shouldn't expose this to an external API

Reply:
yes, good point, I removed it